### DISCUSSION TITLE
Home Assistant OS 10.0.rc3

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#2448] [Announcements]
## Home Assistant Operating System

* Add support for NFS and SMB/CIFS network file systems (#2446) @agners
* Enable Multi-Gen LRU (#2392) @agners
* Use zsmalloc for zswap (#2443) @agners
* Bump buildroot to update NetworkManager to 1.40.16 (#2442) @agners
* Bump buildroot to update Docker to 23.0.2 (#2439) @agners
* Set umask on swapfile creation (#2436) @agners
* Limit systemd-journald log size to 500MB (#2226) (#2435) @agners
* Improve IPv6 NDP for non-local Thread Border routers (#2434) @agners

## Open Virtual Appliance

* Linux: Update kernel 6.1.22 (#2441) @agners

## Generic x86-64

* Linux: Update kernel 6.1.22 (#2441) @agners

## Hardkernel ODROID

* Linux: Update kernel 6.1.22 (#2441) @agners

## ASUS Tinker

* Linux: Update kernel 6.1.22 (#2441) @agners

## Khadas VIM Series

* Linux: Update kernel 6.1.22 (#2441) @agners

## Generic aarch64

* Linux: Update kernel 6.1.22 (#2441) @agners

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/10.0.rc3'>Home Assistant OS 10.0.rc3</a>.</em>

**Comment by cmrcode** [comment]
Excited to try this out! Missing the RPI 4 image?

**Reply by agners** [reply]
Probably was still building when you looked. It is there now.

**Reply by cmrcode** [reply]
Thank you.
